native_client/python/client.py

```diff
@@ -128,14 +128,14 @@ def main():
             ds.setScorerAlphaBeta(args.lm_alpha, args.lm_beta)
 
     fin = wave.open(args.audio, 'rb')
-    fs = fin.getframerate()
-    if fs != desired_sample_rate:
-        print('Warning: original sample rate ({}) is different than {}hz. Resampling might produce erratic speech recognition.'.format(fs, desired_sample_rate), file=sys.stderr)
-        fs, audio = convert_samplerate(args.audio, desired_sample_rate)
+    fs_orig = fin.getframerate()
+    if fs_orig != desired_sample_rate:
+        print('Warning: original sample rate ({}) is different than {}hz. Resampling might produce erratic speech recognition.'.format(fs_orig, desired_sample_rate), file=sys.stderr)
+        fs_new, audio = convert_samplerate(args.audio, desired_sample_rate)
```

Comment:
I guess you can just change this one?

Reply:
Yeah fs_orig can remain fs, I only changed it for improved readability. Could also change "fs_new" to "_" to signify it's ignored, since it's unused.

Reply:
I'd vote for `_`

Reply:
It's not unlikely we made the same mistake on other uses, might be worth checking all our other cases :)

Reply:
Sure thing, I'll look into that and put together another commit

Reply:
@alexcannan gentle ping?

Reply:
Sorry for the wait. To be honest I've been having troubles testing the node js and .net bindings since I haven't ever really used either... I've narrowed their audio length logic down to [here for the javascript client](https://github.com/mozilla/DeepSpeech/blob/8088b574fc4ebeff63c143019f2fa5cd81d9425a/native_client/javascript/client.js#L116) and [here for the .net client](https://github.com/mozilla/DeepSpeech/blob/8088b574fc4ebeff63c143019f2fa5cd81d9425a/native_client/dotnet/DeepSpeechConsole/Program.cs#L89).

The .net client doesn't look like it even converts to 16kHz as far as I can tell. 

The javascript client does, but I'm a bit unsure about its logic, I'm not sure why they divide the audioBuffer length by two. Besides that, this new buffer should be 16kHz as it's the output of the SoX pipe, so the use of the desired_sample_rate var for division should be fine.

Are there any other clients to look at?

Reply:
> Are there any other clients to look at?

No, that's good!